input/system/selfhosted/log_receiver.go

```diff
@@ -386,10 +402,23 @@ func logReceiver(ctx context.Context, server *state.Server, globalCollectionOpts
 				// Note that we need to restore the original trailing newlines since
 				// ProcessLogStream below expects them and they are not present in the tail
 				// log stream.
```

Comment:
Is this still true for syslog?

Reply:
Yes, the way Postgres sends it it does not contain newlines (as Postgres splits at the newline boundary, or 900 characters, whichever is earlier):

https://github.com/postgres/postgres/blob/REL_13_STABLE/src/backend/utils/error/elog.c#L1948